en_US.md

```diff
@@ -2,6 +2,8 @@
 ===
 
 ## What is "996"?
+996 working with you, will be live in ICU.
```

Comment:
how about:
`Working like 996, soon sick at a ICU`

Reply:
Honestly, too complicated.

Reply:
Yeah I feel the above `996 working, ICU waiting` sounds good as well 😆

Reply:
996 working, ICU waiting +1

Reply:
96 working, ICU waiting +1. Great translating